core/store/src/db.rs

```diff
@@ -230,7 +230,18 @@ pub(crate) trait Database: Sync + Send {
     fn transaction(&self) -> DBTransaction {
         DBTransaction { ops: Vec::new() }
     }
-    fn get(&self, col: DBCol, key: &[u8]) -> Result<Option<Vec<u8>>, DBError>;
+
+    fn get(&self, col: DBCol, key: &[u8]) -> Result<Option<Vec<u8>>, DBError> {
+        let start_time = std::time::Instant::now();
+        let result = self.get_internal(col, key);
+        metrics::DATABASE_GET_LATENCY_HIST
+            .with_label_values(&[&format!("{}", col)])
+            .observe(start_time.elapsed().as_micros() as f64);
+        result
```

Comment:
Firstly, use `start_timer` on the metric.  Secondly, use base units in
metrics (so seconds, not microseconds).  Lastly, I suggest we add
`strum::IntoStaticStr` derive on `DBCol` and then use `col.into()` as
label value in metrics.

```rust
         let timer = metrics::DATABASE_GET_LATENCY_HIST
             .with_label_values(&[col.into()])
             .start_timer(...);
         let result = self.get_internal(col, key);
         timer.observe_duration();
         result
```